lib/election.js

```diff
@@ -417,32 +408,48 @@ function _createEventManifest(ledgerNode, blockHeight, callback) {
  *          completes.
  */
 function _recommendElectors(ledgerNode, voter, electors, manifest, callback) {
-  // TODO: use given parameters and blockchain to determine a set of
-  // electors to recommend for the next block
-
-  // FIXME: include all current electors (should filter this by those
-  //   that voted in the last election so we don't recommend those that
-  //   aren't participating)
-  let recommendedElectors = electors.map(elector => ({id: elector.id}));
+  let recommendedElectors = [];
 
   // TODO: implement option to add all nodes that sent in events to the
   // previous block to the elector pool that can be contacted
 
   async.auto({
+    config: callback => _getLatestConfig(ledgerNode, callback),
     latestBlock: callback => ledgerNode.storage.blocks.getLatest(callback),
-    getEventPeers: ['latestBlock', (results, callback) => {
+    latestVoters: ['latestBlock', (results, callback) => {
+      // there are no election results in the genesis block
```

Comment:
This isn't true? There are election results in the genesis block? At least that's what my database says.

Reply:
Maybe what I was seeing was *before* genesis block was written then.

Reply:
@dlongley `ledgerNode.storage.blocks.getLatest`  returns: 
```
eventBlock: {}
```
While working on block 1, since there are no `event` blocks yet.  

Changing the comments.